_components/buttons.md

```diff
@@ -7,7 +7,7 @@ type: element
 title: Buttons
 category: UI components
 lead: Use buttons to signal actions.
-component_url: 'https://components.standards.usa.gov/components/detail/buttons.html'
+fractal_component_url: 'https://components.standards.usa.gov/components/detail/buttons.html'
```

Comment:
One way to make this both less verbose for other components and easier to transition over to making the base fractal URL being supplied by an environment variable would be to set `fractal_component_url: '/components/detail/buttons.html`, and move the `https://components.standards.usa.gov` part of the string into `styleguide.html`, so that it's provided by the template rather than the YAML front matter. It's up to you though, I guess we can also just change it after #357 lands!